### PR TITLE
harvest mef

### PR DESCRIPTION
**Pull request by rerowep**
* NEW: apiharvest module
    * NEW: mef module
    * NEW: persons module for schema and es mapping
    * NEW: automaticly harvest persons from http://mef.test.rero.ch/api/mef

To Test:
```
./scripts/bootstrap
./script/setup
```

Start MEF and ILS server
update source config for mef harvesting locally:

`pipenv run invenio apiharvester source mef http://localhost:5002/api/mef -u`

### CODE REVIEW

File: rero_ils/modules/mef/minters.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+from .providers import MefPersonProvider
+
+
+def mef_person_id_minter(record_uuid, data, pid_key='pid',
+                         object_type='rec'):
+    """RERIOLS Organisationid minter."""
+    assert 'pid' in data
```
Comment (jma) [inline rero_ils/modules/mef/minters.py]: pid_key instead?

File: rero_ils/modules/mef/minters.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Identifier minters."""
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+from .providers import MefPersonProvider
+
+
+def mef_person_id_minter(record_uuid, data, pid_key='pid',
```
Comment (jma) [inline rero_ils/modules/mef/minters.py]: Why not keep MEF pid as local pid?

File: rero_ils/modules/mef/receivers.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+def publish_api_harvested_records(sender=None, records=[], *args, **kwargs):
+    """Create, index the harvested records."""
+    name = kwargs['name']
+    url = kwargs['url']
+    if name == 'mef':
+        converted_records = []
+        jsonschema_host = current_app.config.get('JSONSCHEMAS_HOST', '')
+        for record in records:
+            record['$schema'] = 'http://' + jsonschema_host +\
```
Comment (jma) [inline rero_ils/modules/mef/receivers.py]: add in the config?

File: rero_ils/modules/mef/minters.py (hunk 2 above)
Comment (rerowep) [reply rero_ils/modules/mef/minters.py]: I need a data['pid], I think the assert is correct.

**Pull request merged**